contrib/clojure-package/project.clj

```diff
@@ -0,0 +1,54 @@
+;;
+;; Licensed to the Apache Software Foundation (ASF) under one or more
+;; contributor license agreements.  See the NOTICE file distributed with
+;; this work for additional information regarding copyright ownership.
+;; The ASF licenses this file to You under the Apache License, Version 2.0
+;; (the "License"); you may not use this file except in compliance with
+;; the License.  You may obtain a copy of the License at
+;;
+;;    http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
+
+(defproject org.apache.mxnet.contrib.clojure/clojure-mxnet "1.3.0-SNAPSHOT"
+  :description "Clojure package for MXNet"
+  :dependencies [[org.clojure/clojure "1.9.0"]
+                 [t6/from-scala "0.3.0"]
+
+                 ;; Jars from Nexus
+                 ;[org.apache.mxnet/mxnet-full_2.11-osx-x86_64-cpu "1.2.1"]
```

Comment:
The CI build is taking it from master when the jars are installed locally with `make scalainstall`.  I kept the maven jars lines in there as well commented out so that it is easy for a user to switch back and forth.